### ISSUE TITLE
Publish the latest verion to Maven Central.

### ISSUE THREAD

**Issue by KristinaBurda** [graphql-java/graphql-java-extended-validation#41]
Hello,

Thank you for work on validation library. We(our team) enjoy using it. 

But recently we found out that the latest (16.0.0) version is not available in Maven Central.
Could you please publish the latest version to Maven Central ?

**Comment by bbakerman** [comment]
Whoops that step was missed - synching from Bintray now

**Comment by pukkaone** [comment]
JFrog will be [sunsetting Bintray/JCenter](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/).
- March 31, 2021: No new package versions will be accepted to JCenter.
- May 1, 2021: JCenter API and user interface will no longer be available to non-Artifactory customers.

Are you planning to [migrate to Maven Central](https://blog.sonatype.com/what-publishers-need-to-know-about-migrating-from-jcenter-/-bintray-to-the-central-repository)?

**Comment by bbakerman** [comment]
Yeah - Bintray we a handy way to get to Maven Central - now we will have to
go direct

**Comment by KristinaBurda** [comment]
Hello @bbakerman !

I'm sorry to bother you, I just want to clarify,
Is it any chance to publish the latest version to Maven Central?

Thank you in advance)

**Comment by lkorth** [comment]
As far as I can tell it is now published to maven central and this could be closed.
